src/main/java/jenkins/plugins/git/GitSCMFileSystem.java

```diff
@@ -267,7 +268,13 @@ public boolean supports(SCMSource source) {
         }
 
         @Override
-        public boolean supports(SCMSourceDescriptor descriptor) {
+        public boolean supportsDescriptor(SCMDescriptor descriptor) {
+            // Assume by default that we don't support GitSCM since we can't tell how it would be configured.
+            return false;
```

Comment:
Is this true? Surely if we can ever give yes to GitSCM then we return true. We only return false for other SCMs.

(And iirc any git we can do scmfilesystem by making a local cached clone)

Reply:
Ah, ok, I was just basing my assumption off how `#supports(SCM)` works here.